packages/p2p/src/components/my-profile/payment-methods/payment-methods-list/delete-payment-method-error-modal.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Button, Modal, Text } from '@deriv/components';
+import { observer } from 'mobx-react-lite';
+import { useStores } from 'Stores';
+import { localize } from 'Components/i18next';
+
+const DeletePaymentMethodErrorModal = () => {
+    const { general_store, my_profile_store } = useStores();
+
+    return (
+        <Modal
+            is_open={my_profile_store.is_delete_error_modal_open}
```

Comment:
Hey likhith, could we rename the is_delete_error_modal_open to is_delete_payment_method_error_modal_open? As the current naming isn't clear that we are observing the DeletePaymentMethodErrorModal